velox/exec/WindowPartition.h

```diff
@@ -42,11 +48,39 @@ class WindowPartition {
       const std::vector<std::pair<column_index_t, core::SortOrder>>&
           sortKeyInfo);
 
+  WindowPartition(
```

Comment:
Please add documentation for this constructor as well. This one is used for partial partition when the input data will be a subset of the entire partition.